checks.csv

```diff
@@ -10,3 +10,4 @@
 toe,  [energy],
 tce,  [energy], *
 GW a, [energy],
```

Comment:
Should `GW a` be here? I can't find it defined in `definitions.txt`

Reply:
Yes, it should be here. Pint parses this as 'GW a' = 'GW' × 'a' = ('giga-' × 'watt') × 'year'. It does not require any special configuration because these are all SI units and separated by a space.

If someone (e.g. most MESSAGE instances) insists on using 'Wa' *without* a space between the two units, then the extra definition added in this PR comes into play: 'kWa' = 'kilo-' × 'Wa' = 'kilo-' × ('watt' × 'year').

Reply:
To be clear: having both in checks.csv lets us check that the unit can be parsed either way.